drivers/misc/mcux_flexio/Kconfig

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2023, STRIM, ALC
+# SPDX-License-Identifier: Apache-2.0
+
+config MCUX_FLEXIO
+	bool
+	depends on DT_HAS_NXP_FLEXIO_ENABLED
+	depends on CLOCK_CONTROL
+	help
+	  Enable the FlexIO.
+	  FlexIO is not used separately.
```

Comment:
@danieldegrasse, Do I understand correctly that you are suggesting to add additional information about where this configuration can be configured?

Reply:
That would be fine, yes. I was thinking something like the following (provided that everything in that help string is actually correct):

`Enable the FlexIO controller driver. This driver is not user-selectable, and should be enabled by other FlexIO drivers so that they can use it to share the resources of the FlexIO device`